core/src/main/scala/com/pingcap/tispark/utils/TiUtil.scala

```diff
@@ -207,12 +207,20 @@ object TiUtil {
         conf.get(TiConfigConst.REGION_INDEX_SCAN_DOWNGRADE_THRESHOLD).toInt
       )
     }
+
+    if (conf.contains(TiConfigConst.PARTITION_PER_SPLIT)) {
+      tiConf.setPartitionPerSplit(conf.get(TiConfigConst.PARTITION_PER_SPLIT).toInt)
+    }
+
     tiConf
   }
 
-  def getChunkBatchSize(sqlContext: SQLContext) =
+  def getChunkBatchSize(sqlContext: SQLContext): Int =
     sqlContext.getConf(TiConfigConst.CHUNK_BATCH_SIZE, "1024").toInt
 
+  def getPartitionPerSplit(sqlContext: SQLContext): Int =
+    sqlContext.getConf(TiConfigConst.PARTITION_PER_SPLIT, "10").toInt
```

Comment:
use this value `private static final int DEF_PARTITION_PER_SPLIT = 10;` instead of `"10"`?